docs/proxy/residential_proxy/tips_and_tricks.md

```diff
@@ -5,29 +5,29 @@ paths:
     - proxy/residential-proxy/tips-and-tricks
 ---
 
-# [](#tips-and-tricks)Tips and tricks
+# [](#tips-and-tricks) Tips and tricks
 
-Since Residential proxy is less predictable than datacenter proxy and is priced differently (by number of IPs vs traffic used), there are some important things that need to be considered before you use the Residential proxy in your solutions.
+[Residential]({{@link proxy/residential_proxy.md}}) proxies are less predictable than [datacenter]({{@link proxy/datacenter_proxy.md}}) proxies and are priced differently (by number of IPs vs traffic used). Because of this, there are some important things to consider before using residential proxy in your solutions.
 
-## The traffic used by automated browsers needs to be controlled
+## [](#control-traffic-used-by-automated-browsers) Control traffic used by automated browsers
 
-Since Residential proxy is priced by data traffic used, it's very easy to realy quickly use up all prepaid traffic, especialy when accessing websites with large files loaded on every page.
+Residential proxy is priced by data traffic used. Thus, it's easy to quickly use up all your prepaid traffic. In particular, when accessing websites with large files loaded on every page.
 
-There are two main ways to reduce the traffic use of your solutions:
+There are two main ways to reduce your traffic use.
 
-*   [Block requests which are not required for your solution](https://help.apify.com/en/articles/2423246-block-requests-in-puppeteer)
-*   [Cache responses that you require but do not wish to load again and again](https://help.apify.com/en/articles/2424032-cache-responses-in-puppeteerr)
+*   [Block requests which are not required for your solution](https://help.apify.com/en/articles/2423246-block-requests-in-puppeteer).
+*   [Cache responses that you require but do not wish to load again and again](https://help.apify.com/en/articles/2424032-cache-responses-in-puppeteerr).
```

Comment:
Both of those may or may not be effective at this point and could be bad advice. Both guides use a thing called request interception. The trouble is, request interception disables browser cache in Puppeteer. So in the end, the traffic may actually end up being worse. I think we should do some tests before using those guides. We don't recommend using request interception in the SDK docs and use https://sdk.apify.com/docs/api/puppeteer#puppeteerblockrequests instead.